gocd/pipelines/snuba-sns-test.yaml

```diff
@@ -15,15 +15,28 @@ pipelines:
             GKE_REGION: us-central1
             GKE_CLUSTER_ZONE: a
             GKE_BASTION_ZONE: a
+            MIGRATIONS_READINESS: complete
         group: snuba
         lock_behavior: unlockWhenFinished
         materials:
             snuba_repo:
                 git: https://github.com/getsentry/snuba.git
-                shallow_clone: true
+                shallow_clone: false
```

Comment:
Is this set to false because you need reference to other SHA's?

Reply:
yes